### ISSUE TITLE
Oldversion dependency do not support classifier.

### ISSUE THREAD

**Issue by pethers** [siom79/japicmp#73]
Thanks for providing a compare tool that works with java8, used clirr before.

Currently build jar files with different classifiers that I want to compare, but classifier is not supported inside dependency when configuring old version to compare against.

want to be able to use classifier like below

--
 <oldVersion>
                    <dependency>
                        <groupId>japicmp</groupId>
                        <artifactId>japicmp-test-v1</artifactId>
                        <version>0.5.3</version>
                        <classifier>somevariant</classifier>
                    </dependency>
  </oldVersion>
---

Best regards
Pether

**Comment by siom79** [comment]
Hello Pether,

I have added the requested feature to the develop branch:
```
git clone https://github.com/siom79/japicmp.git
cd japicmp
git checkout develop
mvn clean install
```
Let me know what you think.

**Comment by pethers** [comment]
Excellent, have tested it and works as expected.

**Comment by siom79** [comment]
Released with version 0.6.0.
